File: server/common/bet.py

class Bet:
    def __init__(self, nombre, apellido, dni, nacimiento, numero):
        self.nombre = nombre
        self.apellido = apellido
        self.dni = dni
        self.nacimiento = nacimiento
        self.numero = numero

    def print_bet(self):
        print(f"Nombre: {self.nombre}")
        print(f"Apellido: {self.apellido}")
        print(f"DNI: {self.dni}")
        print(f"Nacimiento: {self.nacimiento}")
        print(f"Numero: {self.numero}")

File: client/common/bet.go
package common

type Bet struct {
	Nombre        string
	Apellido      string
	DNI           string
	Nacimiento    string
	Numero        string
	Agencia		  string
}

const BET_SUCCESS = 0
const BET_FAIL = 1

const BET_RECORD_LENGTH = 5